docs/source/install.rst

```diff
@@ -1,7 +1,7 @@
 Install
 *******
 
-Featuretools is available for Python 3.7 and 3.8. The recommended way to install Featuretools is using ``pip`` or ``conda``::
+Featuretools available for Python 3.7, 3.8, and 3.9. It can be installed from PyPI, conda, or from source.::
```

Comment:
I think we want to keep the "is" in the first sentence.

The period is unnecessary and when built reads like this:

> It can be installed from PyPI, conda, or from source.:

Should it be Anaconda instead of conda if we're saying PyPI instead of pip?

I feel like users will be a bit more familiar with pip and conda compared to PyPI and Anaconda

Reply:
Fixed to say pip